src/test/java/christmas/badge/BadgeTest.java

```diff
@@ -0,0 +1,19 @@
+package christmas.badge;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+
+class BadgeTest {
+
+    @ParameterizedTest
```

Comment:
`@ParameterizedTest`를 사용하면서도 테스트의 이름을 설정해 줄 수 있답니다!
각각의 파라미터 케이스에 대해 이름이 적용됩니다!
```suggestion
    @ParameterizedTest(name = "총 주문 금액이 {0}원인 경우 {1} 뱃지를 얻게 된다.")
```